courses/factories.py

```diff
@@ -1,13 +1,18 @@
-# pylint: disable=missing-docstring
+"""Factories for testing"""
 import factory
 from factory.django import DjangoModelFactory
 from .models import Module, Course
 
+
 class CourseFactory(DjangoModelFactory):
-    class Meta:
+    """Factory for Course"""
```

Comment:
I agree with George for this reason: Decisions take time, even trivial decisions.  At it's worst, this rule makes you add a two word docstring that adds no information.  At it's best, it keeps us from having to decide and then discuss whether a docstring is useful or not.